packages/oas/test/reducer/index.test.ts

```diff
@@ -17,16 +15,6 @@ import reduceQuirks from '../__datasets__/reduce-quirks.json';
 import securityRootLevel from '../__datasets__/security-root-level.json';
 import tagQuirks from '../__datasets__/tag-quirks.json';
 
-declare global {
-  interface Console {
-    logx: any;
-  }
-}
-
-console.logx = (obj: any) => {
```

Comment:
lol wut

Reply:
this snuck in in a commit i made this morning where i added that skipped failing test. `console.logx` is a tool i use to dump full objects.